common/test/acceptance/pages/lms/instructor_dashboard.py

```diff
@@ -773,127 +774,182 @@ def is_browser_on_page(self):
         return self.q(css='a[data-section=student_admin].active-section').present
 
     @property
-    def student_email_input(self):
+    def entrance_exam_student_email_input(self):
```

Comment:
Are you still getting `invalid-name` pylint errors? It's probably because some of these names are getting long. You can append `# pylint: disable=invalid-name` to the line to prevent these errors. In general, I think we tend to avoid abbreviated names where possible in our platform.